test/k8sT/Chaos.go

```diff
@@ -15,6 +15,7 @@
 package k8sTest
```

Comment:
Commit message typos: 

"Run netperf in background while Cilium pod is being deleted"

"two new tests were added"

"One tests that the connectivity from client to server is not lost without any policy"